pkg/kudoctl/cmd/generate/testdata/maintainer.golden

```diff
@@ -37,17 +37,29 @@ tasks:
 - kind: Apply
   name: infra
   spec:
+    appVersion: ""
```

Comment:
Does that mean that now every task needs to set these values? We should ignore empty values instead, especially for tasks that aren't "KudoOperator" tasks. How about adding `omitempty` tags to the `KudoOperatorTaskSpec`?

Reply:
Good catch! 👍